algorithms/src/snark/marlin/marlin.rs

```diff
@@ -234,6 +234,8 @@ where
         .map_err(SNARKError::from)
     }
 
+    // Prove that the verifying key indeed includes a part of the reference string,
+    // as well as the indexed circuit (i.e. the circuit as a set of linear-sized polynomials).
```

Comment:
Let's make these doc comments.

```suggestion
    /// Prove that the verifying key indeed includes a part of the reference string,
    /// as well as the indexed circuit (i.e. the circuit as a set of linear-sized polynomials).
```

Reply:
Adjusted